src/components/ContactUs.jsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+const ContactUs = ({ data }) => {
+  const classes = useStyles();
+
+  return (
+    <div data-cy="contact-us" className={classes.container} style={styles.container}>
```

Comment:
I think better change to Box element instead of div, so we have consistent code style

Reply:
fixed 👍